### DISCUSSION TITLE
Way to specify logo URL

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/edit.tosdr.org#1036] [Ideas]
The most recent additions on https://tosdr.org/ have not logo in `https://tosdr.org/logo/<name>.png`
If the phoenix user specifies an image URL, then the build script can convert this to a png of the right size.

We could default to https://domain.com/favicon.ico, although that's probably not a good image resolution

**Comment by evilcel3ri** [comment]
So the image is coming from the user ? Isn't there a way to automate the logo from Wikipedia or something like it ? Are we hosting the image or is it remote ?

**Comment by michielbdejong** [comment]
We can use favicons or wikipedia or web search as a suggestion, but I don't think we can automate it 100% (unless you know of a good website logo database we can import from?), so I think a phoenix user should still select the image to display.
But the current situation is that there is no way at all to influence the logos we display from within phoenix.

Most logos we have now come either from Hugo or from me, using web search and some manually applied image resize tools.

We host them, they are in https://github.com/tosdr/tosdr.org/tree/master/logo
